crates/sui-framework/sources/object.move

```diff
@@ -4,8 +4,6 @@
 /// Sui object identifiers
```

Comment:
This file contains the important Move changes. Everything else is just shifting to fit the API changes here.